index.js

```diff
@@ -103,22 +91,23 @@ WuiDom.prototype._assign = function (tagName, options) {
 		}
 	} else if (tagName instanceof window.Element) {
 		// the first passed argument already is a real HTML Element
-
 		this.rootElement = tagName;
 	} else {
 		throw new Error('WuiDom.assign requires the given argument to be a DOM Element or tagName.');
 	}
 
-	if (options && options.name) {
-		this._name = options.name;
-	}
+	options = options || {};
 
-	if (options && options.hidden) {
-		// start hidden
+	// start hidden
+	if (options.hidden) {
 		this.hide();
 	}
 
-	return this.rootElement;
+	// set identifier (used by getChild)
+	this._name = String(options.name);
```

Comment:
Yes and No. (I forgot to push the latest change)
I let the developer take the responsibility to not set `name` to undefined